packages/xrpl/src/models/methods/accountObjects.ts

```diff
@@ -1,35 +1,7 @@
-import {
-  AMM,
-  Bridge,
-  Check,
-  DepositPreauth,
-  Escrow,
-  Offer,
-  PayChannel,
-  RippleState,
-  SignerList,
-  Ticket,
-  XChainOwnedClaimID,
-  XChainOwnedCreateAccountClaimID,
-} from '../ledger'
+import { LedgerEntryFilter, LedgerEntry } from '../ledger'
 
 import { BaseRequest, BaseResponse, LookupByLedgerRequest } from './baseMethod'
 
-export type AccountObjectType =
```

Comment:
I find having a subset type helpful for autocomplete and type checking for example if I try `account_objects` with `fee` I would like typescript to fail.

Reply:
This shows how to do this with fun fruits. https://fjolt.com/article/typescript-exclude-type

Reply:
rippled actually is fine with using `fee` in `account_objects` (Clio complains though)